src/ansys/fluent/core/services/field_data.py

```diff
@@ -838,6 +838,155 @@ def _extract_field(field_datatype, field_size, chunk_iterator):
     return fields_data
 
 
+class BaseFieldData:
+    """Contain common properties required by all field data types."""
+
+    def __init__(self, i_d, data):
+        self._data = data
+        self._id = i_d
+
+    @property
+    def data(self):
+        return self._data
+
+    @property
+    def surface_id(self):
+        return self._id
+
+    @property
+    def size(self):
+        return len(self._data)
+
+    def __getitem__(self, item):
+        return self._data[item]
+
+
+class ScalarFieldData(BaseFieldData):
+    """Container for scalar field data."""
```

Comment:
```suggestion
    """Provides a container for scalar field data."""
```
We want class descriptions to start with a verb ending in "s" or "es" so that an autosummary table is formatted nicely (and consistently).